### ISSUE TITLE
Service may not be available

### ISSUE THREAD

**Issue by chiragchhita** [vidanov/lambda-nodejs-lightsail-backup#5]
Hi,
I am trying to use your script to setup Light sail backups, but seem to be running into an issue.
The Test Event works, but within that test event, I see the below error.
I have also tested this script on another Lightsail instance that is in us-east-2, with the same issue.

I primarily use US-EAST-1, and have created this lambda script in both US-EAT-1 and US-EAST-2 to see if that was the issue.

> UnknownEndpoint: Inaccessible host: `lightsail.ap-southeast-2a.amazonaws.com'. This service may not be available in the `ap-southeast-2a' region

Any idea of what the issue could be?
Thanks!

**Comment by vidanov** [comment]
I do not know. I did not have this issue. I used Frankfurt for me, but it should work with any other region too.
Try to double check if you setup the function in the same region where you have your LightSail instance
http://take.ms/ZEq31  Prove the name of your instance too.
https://docs.aws.amazon.com/general/latest/gr/rande.html
Check the permissions
http://take.ms/V82PE it should be like this in the Lambda
